src/main/java/net/minecraftforge/gradle/common/BaseExtension.java

```diff
@@ -323,7 +323,8 @@ else if (rightMc && !rightChannel)
         throw new GradleConfigurationException("The specified mapping '" + getMappings() + "' does not exist!");
     }
 
-    private static String getMappedString(String key, String value) {
+    @SuppressWarnings("unused")
+	private static String getMappedString(String key, String value) {
```

Comment:
ForgeGradle uses spaces to indent, not tabs

Reply:
Whoops must've been eclipse that messed up. Will fix in a minute.

Reply:
Worth noting that eclipse has a button to show whitespace (I'm 90% sure that it's a ¶), which might be useful in the future.

Reply:
Yeah that could be useful 🤔 
I thought I'd corrected all the automatic formatting/indentation but I guess I forgot this part.